src/Symfony/Component/Routing/Generator/UrlGenerator.php

```diff
@@ -295,6 +295,13 @@ protected function doGenerate(array $variables, array $defaults, array $requirem
             return $a == $b ? 0 : 1;
         });
 
+        array_walk_recursive($extra, static function ($value) {
```

Comment:
I think $value should be a reference and the function shouldn't return

Reply:
Indeed, thanks